src/Common/src/Interop/Windows/kernel32/Interop.GetFileAttributesEx.cs

```diff
@@ -68,10 +67,8 @@ internal unsafe struct WIN32_FIND_DATA
             internal uint nFileSizeLow;
             internal uint dwReserved0;
             internal uint dwReserved1;
-            [MarshalAs(UnmanagedType.ByValTStr, SizeConst = 260)]
-            internal string cFileName;
-            [MarshalAs(UnmanagedType.ByValTStr, SizeConst = 14)]
-            internal string cAlternateFileName;
+            internal FixedString.Size260 cFileName;
+            internal FixedString.Size14 cAlternateFileName;
```

Comment:
> I don't agree.

Then you and I will need to agree to disagree.  I would strongly prefer not using these custom structs.  But it's up to you.

> I also don't buy that looking at well-documented struct inside a P/Invoke struct is that onerous for a well-versed C# programmer.

I believe I am a well-versed C# programmer, and they are onerous for me.